test/functional/apps/discover/group3/_lens_vis.ts

```diff
@@ -616,7 +615,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       expect(await getCurrentVisTitle()).to.be('Pie');
       await testSubjects.existOrFail('partitionVisChart');
 
-      await discover.chooseLensSuggestion('barVerticalStacked');
+      await discover.chooseLensSuggestion('heatMap');
```

Comment:
Hi @mbondyra,

Thanks for looking into it!

I think the idea of this test was to select another type plus apply a customization like on the line below `await changeVisShape('Line');`.

The `heatMap` would not have `Line` as an option. Can we pick something which can be also customized easily here in tests?

Reply:
I initially thought that's the case too, but actually these two actions do not have to share a common 'parent' chart type to test what's assumed in the description of this test. What we do in these 2 steps is:
1. Click on the suggestion and apply it
2. Open the flyout again and then change the chart type from the chart switcher (can be any available type there).

We lost the access to the bar/line/area chart from the discover suggestions with the change from the[ pr I linked](https://github.com/elastic/kibana/pull/197101), so I chose another one, but if it's important to stick to one parent, I've changed to waffle -> treemap. Testing right now and will push once I see it's passed.